src/vtol.py

```diff
@@ -65,13 +65,28 @@ def __init__(self, *args): #pylint: disable=useless-super-delegation
     heading = None
     MISSION_COMPLETED = False
     coms = None
-
     # pylint: disable=no-self-use
-    def coms_callback(self, message, _):
+    def coms_callback(self, message):
         '''callback for radio messages'''
-        # TODO respond to xbee messagge
-        data = json.loads(message.data)
-        print(data['type'])
+        parsed_message = json.loads(message.data)
+        #tuple of commands that can be executed
+        valid_commands = ("takeoff", "RTL")
```

Comment:
Cool, this is fine for now, but in the future lets move this to the config file, once this is more flushed out with GCS